pkg/storage/filesystem_test.go

```diff
@@ -0,0 +1,32 @@
+package storage
+
+import (
+	"testing"
+)
+
+func TestGetFileSystems(t *testing.T) {
+	fstypes, err :=  internalGetFilesystems("tests/filesystems")
+	expected := [3]string{"ext4", "ext3", "vfat"}
+	if err != nil{
+		t.Error("Failed to open the proc filesystems")
+	}
+	if len(fstypes) != len(expected){
+		t.Errorf("Expected '%v', but resulted with '%v'", expected, fstypes)
+	}
+	for index, value := range fstypes{
+		if value != expected[index]{
+			t.Error("Wrong filesystems appened.")
+		}
+	}
+}
+
+
+func TestEmptyFileSystems(t *testing.T) {
+	fstypes, err := internalGetFilesystems("tests/emptyFile")
+	if err != nil{
+		t.Error("Failed to open the proc filesystems")
+	}
+	if len(fstypes) != 0{
+		t.Error("The file is supposed to be empty.")
```

Comment:
Maybe change this to explain why you expect that, something like `t.Error("Expected no results for empty filesystem file")`?